_posts/2017-10-20-the-data-and-science-behind-grabshare-part-i.md

```diff
@@ -0,0 +1,117 @@
+
+![constraint](/img/the-data-and-science-behind-grabshare-part-i/constraint.png)
+
+where s(drj) denotes the total capacity of the vehicle drj, op(drj) is one of the maintained variable that denotes the current occupied capacity of the vehicle drj and rp (bki) is the required capacity for booking bki. To make it consistent, we also need to update the vehicle occupied capacity variable op (drj) by adding the booking required capacity rp (bki) after every successful assignment or removing it if cancellation occurs.
+
+![GrabShare booking assignment flow](/img/the-data-and-science-behind-grabshare-part-i/grabshare-booking-assignment-flow.png)
+*Algorithm II. GrabShare booking assignment flow*
+
+![GrabShare match case in Singapore](/img/the-data-and-science-behind-grabshare-part-i/grabshare-match-case-singapore.png)
+*Figure 2. GrabShare match case in Singapore*
+
+Secondly, GrabShare’s user experience is different from GrabCar due to the sharing concept. Here we defined some measures to evaluate the GrabShare matching, taking into consideration the trip angle, eta (short for Expected Time of Arrival), detour and efficiency. These measures are used to exclude unacceptable matches and to quantify how good the match is. For example, given a matching route scenario of two bookings (n = 1) as shown in Figure 2. At the first step the driver receives the first GrabShare booking from point A to D (25 minutes direct trip time). After the driver picks up the first passenger and reaches location B on his way to D, he/she is assigned to pickup the second booking from C to E (21 minutes direct trip time). A GrabShare match happens and the final route sequence is generated as A→B→C→D→E. With pooling, it takes 29.5 minutes for the first passenger and 27 minutes for the second passenger to reach their destinations, respectively. Overall it is a good match as the passengers are only delayed a little bit by pooling with a promising driver utilization rate. In this case the driver only needs to drive 23.72km in total to serve two bookings, instead of a total of 39.13km if they were served separately. Not only does this allow passengers to be allocated rides, but drivers save considerable time and money through this efficiency, while increasing their earning power simultaneously. 
+
+This is ultimately deemed a good match, but the details on how we quantify this and its corresponding optimisation model are explained in **Part II**. 
+
+### References 
+
+<a name="1" href="#1">[1]</a>  Grab, "Grab extends grabshare regionally with malaysias first on-demand carpooling service," 2017. [Online]. Available: <https://www.grab.com/my/press/business/grabsharemalaysia/>
```

Comment:
Are you trying to achieve footnotes with this? There's already a way to do footnotes. Refer to [this post](http://engineering.grab.com/a-key-expired-in-redis-you-wont-believe-what-happened-next)

Reply:
okay, will try. i didn't find the post you linked so i went to invent my own way 😂

Reply:
<img width="454" alt="screen shot 2017-10-19 at 6 22 04 pm" src="https://user-images.githubusercontent.com/1209810/31766547-6e5ad64a-b4fa-11e7-87b7-ede16e3e38f3.png">
it looks weird when rendered in this case because we are writing in a scientific style, so let's stick to anchors?

Reply:
Alright. The amount of custom HTML required still scares me 😱